### PR TITLE
Add new user to the owners list

### PR DESCRIPTION
**Pull request by oriAdler**

### CODE REVIEW

**Comment by oriAdler** [comment]
@zgalor @gdbranco PTAL

File: OWNERS [hunk 1]
```diff
@@ -9,6 +9,7 @@ reviewers:
 - oriAdler
 - gdbranco
 - renan-campos
+- tzvatot
```
Comment (gdbranco) [inline OWNERS:12]: I think we actually need to extra line

**Comment by vkareh** [comment]
@tzvatot

> User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

You need to join the OpenShift org.

**Pull request closed without merging**